roles/kubernetes/preinstall/templates/openstack-cloud-config.j2

```diff
@@ -7,3 +7,12 @@ tenant-id={{ openstack_tenant_id }}
 {% if openstack_domain_name is defined and openstack_domain_name != "" %}
 domain-name={{ openstack_domain_name }}
 {% endif %}
+
+{% if openstack_lbaas_enabled %}
```

Comment:
`{% if openstack_lbaas_enabled and openstack_lbaas_subnet_id %}`
Both must be set. Empty subnet id will result in a failure.